pmm-app/src/shared/styles.scss

```diff
@@ -174,3 +174,11 @@ ol.breadcrumb-container {
 .app-tooltip:hover::before {
   display: inline-block;
 }
+
+.custom-grafana-plugin {
```

Comment:
Sorry, this shall not be merged to the trunk. This breaks the grafana layout, we should not mess with styles outside of the panel.

Reply:
Done!